README.md

```
@@ -150,48 +174,62 @@ I'm using InsForge as my backend platform, call InsForge MCP's fetch-docs tool t
+curl -fsSL https://raw.githubusercontent.com/InsForge/InsForge/main/deploy/setup.sh | sh -s ~/project1
+curl -fsSL https://raw.githubusercontent.com/InsForge/InsForge/main/deploy/setup.sh | sh -s ~/project2
+```
+
+Then give each a project name and its own ports. Both `.env` files start with
+`COMPOSE_PROJECT_NAME=insforge`, and **two directories sharing that name share
+containers** — the second `up -d` adopts the first's, rebuilt with the second's
+config. Set it before starting anything.
+
+`~/project1/.env`:
+
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)